vips.h

```diff
@@ -283,6 +283,12 @@ vips_is_16bit (VipsInterpretation interpretation) {
 
 int
 vips_flatten_background_brigde(VipsImage *in, VipsImage **out, double r, double g, double b) {
+	if (vips_is_16bit(in->Type)) {
+		r = r/255 * 65535;
```

Comment:
Might not be a significant issue, but you can get rid of potential inaccuracy from decimal math by doing the multiplication first: `r = 65535 * r/255`. Also possible that the compiler automatically does this, not sure.